src/wp-includes/cache.php

```diff
@@ -131,6 +131,35 @@ function wp_cache_set_multiple( array $data, $group = '', $expire = 0 ) {
 
 	return $wp_object_cache->set_multiple( $data, $group, $expire );
 }
+/**
+ * Removes all cache items in a group.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_Object_Cache::flush_group()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param string|array $group name(s) of group to remove from cache.
+ *
+ * @return bool True on success, false on failure group not found.
+ */
+function wp_cache_flush_group( $group ) {
```

Comment:
This functions needs a fallback in `cache-compat.php` like all other newly introduced cache functions, so that object caches not implementing it still work.

Reply:
OK added it